cli/azd/pkg/devcenter/platform.go

```diff
@@ -45,7 +45,7 @@ func (p *Platform) IsEnabled() bool {
 // ConfigureContainer configures the IoC container for the devcenter platform components
 func (p *Platform) ConfigureContainer(container *ioc.NestedContainer) error {
 	// DevCenter Config
-	container.MustRegisterSingleton(func(
+	container.MustRegisterTransient(func(
```

Comment:
Since the ADE config is now registered as a transient we must update the consumers of the config as well so that a new fresh config can be constructed with all the updated metadata.